common/tls/src/main/java/io/helidon/common/tls/ConfiguredTlsManager.java

```diff
@@ -59,10 +53,12 @@ public class ConfiguredTlsManager implements TlsManager {
 
     private final String name;
     private final String type;
-    private final List<TlsReloadableComponent> reloadableComponents = new CopyOnWriteArrayList<>();
-    private final AtomicReference<X509KeyManager> keyManager = new AtomicReference<>();
-    private final AtomicReference<X509TrustManager> trustManager = new AtomicReference<>();
-    private volatile boolean enabled;
+
+    private volatile X509KeyManager keyManager;
```

Comment:
Why do we have to carry two of each? Seems conceptual to only carry a single one that may or may not be reloadable. Applies to the TrustManager also.

Reply:
We cannot reload using a reloadable component (or we should not). The reason is to be able to provide the unwrapped X509 manager used to create this instance, and to have the reloadable instance, so we can reload when requested from outside (or from a descendant).